CHANGELOG.md

```diff
@@ -17,8 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added optional props to specify icon titles in the `AudioInputControl` and `ContentShareControl` components.
 - Added optional props to specify the dropdown text that shows when no video quality is selected, in the `QualitySelection` component.
-
+- Added observer to `selectAudioInputDeviceError` and `selectVideoInputDeviceError` to deliver the error from SDK level to client application level.
```

Comment:
Could you please change this to "Add"? My bad that I didn't check the last two changelogs to follow this. But general good practice is to keep it in the present tense.

Reply:
Sure, should I change all of the 'Added' to 'Add'?

Reply:
Actually never mind, you can just change yours for now. We have a lot of "Added" elsewhere anyways 😅 .

Reply:
I really want to fix all tense issues at a same time, lol